docusaurus/docs/getting-started.mdx

```diff
@@ -22,44 +24,54 @@ Plugin tools consists of two packages:
   queryString="current-package-manager"
 />
 
-Follow the prompts to scaffold a Grafana plugin. [See here](./creating-a-plugin.mdx) for detailed information about creating a plugin.
+For next steps, see [Creating a plugin](./creating-a-plugin.mdx) 
 
 :::note
 
-Plugins previously scaffolded with `@grafana/toolkit` can be migrated. Please see [this migration guide](./migrating-from-toolkit).
+If you have previously built a plugin with `@grafana/toolkit`, you can use our plugin tools to simplify migration. For more information, refer to the [Migration guide](./migrating-from-toolkit).
 
 :::
 
-### What you'll need
+### Before you begin 
 
-- [Go](https://go.dev/doc/install) version 1.18 or above
-- [Mage](https://magefile.org/)
-- [Node.js](https://nodejs.org/en/download/) version 16 or above:
-  - When installing Node.js, you are recommended to check all checkboxes related to dependencies.
-- Optionally [Yarn 1](https://classic.yarnpkg.com/lang/en/docs/install) or [PNPM](https://pnpm.io/installation)
-- [Docker](https://docs.docker.com/get-docker/)
+#### Supported operating systems
 
-### Supported Operating Systems
-Create Plugin currently supports the following operating systems:
+Grafana plugin tools work with the following operating systems:
 
 - Linux
 - macOS
```

Comment:
Are there particular versions of Linux or macOS? We list a Windows version.

Reply:
Resolving as postponed. This is something we should specify; however, in phase 1 of this project I am focusing on style and readability issues only. In phase 3, I will gather additional details. That said, if someone on the plugins team sees this comment and wants to contribute an answer now, please do so.

Reply:
>Are there particular versions of Linux or macOS? We list a Windows version.

We list a windows version because I don't believe any version of Windows prior to 10 had WSL. I'm unaware of anything in the tooling that would impact it running on different versions of Linux or Mac OS so long as the OS is compatible with Node 16+.